phoenix-core-client/src/main/java/org/apache/phoenix/util/CDCUtil.java

```diff
@@ -89,7 +89,7 @@ public static String makeChangeScopeStringFromEnums(Set<PTable.CDCChangeScope> i
   }
 
   public static String getCDCIndexName(String cdcName) {
-    return CDC_INDEX_PREFIX + SchemaUtil.getTableNameFromFullName(cdcName);
+    return CDC_INDEX_PREFIX + cdcName;
```

Comment:
I think this is still not being used i.e. the cdc index getting created still follows the format of `CDC_INDEX_PREFIX + SchemaUtil.getTableNameFromFullName(cdcName)`

Reply:
one test in TableTTLIT was passing schema name as well, we should only be passing CDC Object's name everywhere

Reply:
What is the name of the index used if the table name is: `test.table1.table2`? Here, `test` is schema name and table name is `table1.table2`